website/react-magma-docs/src/pages/api/theme.mdx

```diff
@@ -30,20 +30,9 @@ import {
 
 const magmaDark = Object.assign({}, magma, {
   colors: Object.assign({}, magma.colors, {
-    primary: magma.colors.secondary600, // link color
-    focus: '#FFFFFF',
-    danger: magma.colors.danger200,
-    neutral: '#DFDFDF',
-    neutral100: '#000000',
-    neutral200: '#F7F7F7',
-    neutral300: '#DFDFDF',
-    neutral400: '#BFBFBF',
-    neutral500: '#8f8f8f',
-    neutral600: '#707070',
-    neutral900: '#575757',
-    neutral800: '#3F3F3F',
-    neutral700: '#ffffff',
-    border: magma.colors.secondary600,
+    primary: `${magma.colors.tertiary500}`,
+    focus: `${magma.colors.focusInverse}`,
+    danger: `${magma.colors.danger500}`,
```

Comment:
These changes should be reversed. @greenervigil recently updated this correctly in https://github.com/cengage/react-magma/pull/723/files#diff-c8fb829e210501c88af80b591bba017744a762c34bcee8b5e55c3628714b5f33R33-R46.